src/main/kotlin/no/nav/familie/ef/mottak/repository/SøknadRepository.kt

```diff
@@ -16,7 +16,14 @@ interface SøknadRepository :
 
     fun findByJournalpostId(jounalpostId: String): Søknad?
 
-    fun findAllByFnr(fnr: String): List<Søknad>
+    @Query(
+        """
+        WITH q AS (SELECT s.*, ROW_NUMBER() OVER (PARTITION BY dokumenttype ORDER BY opprettet_tid DESC) rn 
+                   FROM soknad s WHERE fnr=:fnr)
+        SELECT * FROM q WHERE rn = 1
```

Comment:
```suggestion
                SELECT * FROM (SELECT s.*, ROW_NUMBER() OVER (PARTITION BY dokumenttype ORDER BY opprettet_tid DESC) rn 
                   FROM soknad s WHERE fnr=:fnr)
   WHERE rn = 1
```

Reply:
🙈 helt enig, endrer til å bruke subquery i stedet for With